pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -184,6 +185,17 @@ def implements(self, interface=None):
     
     def name(self):
         return self.opts.get('name', None)
+
+    def setClickable(self, s, width=None):
+        self.curve.setClickable(s, width)
+
+    @property
+    def clickable(self):
+        return self.curve.clickable
+
+    @clickable.setter
```

Comment:
Good point, I hadn't thought about that.  My goal was to mirror the interface of `PlotCurveItem` so that users don't need to know about the underlying implementation in order to set that property outside the initialization. Especially since the `clickable` property is `False` by default.

An alternative would be to have line 177 read:
```
self.curve.setClickable(kargs.get('clickable', False))
```
if you prefer.  That way the example would still work interchangably with `PlotDataItem`, but if someone is keeping track of what's returned from `PlotItem.plot()` they will have to know to use the `curve` to change that attribute.